### PR TITLE
consume useModal hook

### PR DESCRIPTION
**Pull request by avivtur**
switch to `useModal` hook instead of using a state for each modal

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

File: src/views/catalog/wizard/tabs/disks/components/DiskRowActions.tsx [hunk 1]
```diff
@@ -18,25 +19,15 @@ type DiskRowActionsProps = {
...
 const DiskRowActions: React.FC<DiskRowActionsProps> = ({ diskName }) => {
-  const { vm, updateVM } = useWizardVMContext();
-
   const { t } = useKubevirtTranslation();
+  const { vm, updateVM } = useWizardVMContext();
+  const { createModal } = useModal();
   const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
-  const [isDeleteModalOpen, setIsDeleteModalOpen] = React.useState(false);
+
+  // const [isDeleteModalOpen, setIsDeleteModalOpen] = React.useState(false);
```
Comment (glekner) [inline src/views/catalog/wizard/tabs/disks/components/DiskRowActions.tsx:27]: comment

**Review by glekner — COMMENTED** [review]
awesome!

File: src/views/catalog/wizard/tabs/disks/components/DiskRowActions.tsx (hunk 1 above)
Comment (avivtur) [reply src/views/catalog/wizard/tabs/disks/components/DiskRowActions.tsx]: thanks fixed

**Pull request merged**